### ISSUE TITLE
Cannot apply mna:wind_wall as ActiveEffect

### ISSUE THREAD

**Issue by SargeRyong** [Mithion/Mana-And-Artifice#626]
**Describe the bug**
This isn't a bug report precisely, but I've been trying to get this to work in some custom scripting for my modpack and after a great deal of frustration, I was hoping you'd be able to point me in the right direction for getting this to work.

I'm trying to summon a mob that has an mna spell effect (in this case, wind_wall) applied to it.  "/effect give \<entity\> \<sec\> \<amp\>" works totally fine.  However, "/summon \<entity\> ~ ~ ~ {ActiveEffects:[{"forge:id":"mna:wind_wall",Duration:\<seconds\>,Amplifier:\<amp\>}]}" does not apply the effect for some reason (and yet it works if I substitute in other modded effects).

I initially thought I'd dig into the source to try and see why this wasn't working (maybe it doesn't work like most effects?) before realizing the source was not openly available, so I'm hopeful that I can find some help here instead.

**To Reproduce**
1. Execute command to summon an entity with a different effect (feel free to substitute any valid effect from any namespace here). E.g.:
```
/summon minecraft:iron_golem ~ ~ ~ {ActiveEffects:[{"forge:id":"minecraft:strength",Duration:20000,Amplifier:99b}]}
```
2. Observe that the effect is applied as expected, then kill or otherwise remove the summoned entity.
3. Execute command to summon entity with the wind wall effect.  E.g.,:
```
/summon minecraft:iron_golem ~ ~ ~ {ActiveEffects:[{"forge:id":"mna:wind_wall",Duration:20000,Amplifier:99b}]}
```
4. Observe that the effect is not present.
5. Kill or otherwise remove the summoned entity and summon another one without the ActiveEffect NBT this time.
6. To verify that the effect does work when applied via other means, execute:
```
/effect give @e[type=minecraft:iron_golem,limit=1,sort=nearest] mna:wind_wall 20000 99
```
7. Observe that the effect is applied as expected.
8. Run the following command to observe the correctly affected entity's NBT:
```
/data get entity @e[type=minecraft:iron_golem,limit=1,sort=nearest] ActiveEffects
```
10. Observe that this NBT agrees with the failing NBT used summon the previous entity

**Expected behavior**
I'd like to be able to apply the effect to an arbitrary entity on summon.  I realize this is likely not a concern of the author, but help would be greatly appreciated!

**Version Information:**
 - MC 1.18.2
 - Forge 40.2.0
 - MNA 1.7.2.10

**Comment by Mithion** [comment]
I'll admit I have never actually tried to apply an effect in that way!

It is actually applying, but wasn't looking like it for 2 reasons:

1.) The PotionApplyingEvent isn't fired when you spawn the entity with the effects pre-added.  This event is needed to set up the flags and send the sync packets to turn on the particles for the effect.  There isn't much I can do here.

2.) The wind wall effect wasn't able to function on non-players.  I've made a change for you and next version it will work on living entities!

For reference, I tested with the following command:
/summon minecraft:iron_golem ~ ~ ~ {ActiveEffects:[{"forge:id":"mna:wind_wall",Duration:2000,Amplifier:1}]}

and they reflected arrows I fired back at me with the change.  I hope this helps!
